NewBank/newbank/server/NewBankClientHandler.java

```diff
@@ -34,6 +39,12 @@ public void run() {
 			// if the user is authenticated then get requests from the user and process them 
 			if(customer != null) {
 				out.println("Log In Successful. What do you want to do?");
+
+				out.println("1. Check your accounts");
+				out.println("2. Open new Main account");
+				out.println("3. Open new Savings account");
+				out.println("4. Open new Checking account");
+
```

Comment:
I think it should say specifically that you have to type '1', or '2' etc, to make it a bit easier to know what to do